lib/instruments.js

```diff
@@ -149,8 +150,8 @@ class Instruments {
     this.proc.stderr.setEncoding('utf8');
     let actOnStderr = (output) => {
       if (this.launchTimeout.afterSimLaunch && output && output.match(/CLTilesManagerClient: initialize/)) {
-        this.addSocketConnectTimer(this.launchTimeout.afterSimLaunch, 'afterLaunch', async () => {
-          await killAllInstruments();
+        this.addSocketConnectTimer(this.launchTimeout.afterSimLaunch, 'afterLaunch', async (done) => {
```

Comment:
Actually we only setExitListener after timeout occurs. We always expect the instruments will be killed and do next action (call done event).